broker/patron_request/service/action.go

```diff
@@ -135,8 +138,23 @@ func (a *PatronRequestActionService) updateStateAndReturnResult(ctx common.Exten
 }
 
 func (a *PatronRequestActionService) validateBorrowingRequest(ctx common.ExtendedContext, pr pr_db.PatronRequest) (events.EventStatus, *events.EventResult) {
-	// TODO do validation
-
+	if !pr.Tenant.Valid {
+		return events.LogErrorAndReturnResult(ctx, "missing tenant", nil)
+	}
+	symbols := []string{pr.Tenant.String}
```

Comment:
The raw `tenant` value must be mapped to the symbol first (e.g `ISIL:AU-{tenant}`. But, I think, tenant validation and mapping tenant-to-symbol should happen when the patron request is created, not here. Here we should use the `requester_symbol` value.